grpc_client_test.go

```
@@ -31,26 +29,56 @@ func TestGetStreamYouTube404(t *testing.T) {
+
+	t.Run("falls back for unknown error suffix", func(t *testing.T) {
+		got := streamResolveError("500 - Unknown error", "failed to get stream")
+		if got.Error() != "failed to get stream" {
+			t.Fatalf("unexpected message: %q", got.Error())
+		}
+	})
+}
+
+func TestStreamResolveErrorIsUsableAsNotice(t *testing.T) {
+	err := streamResolveError("Channel 'offlineuser' is offline", "stream not found or offline")
+	if errors.Is(err, errors.New("stream not found or offline")) {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)